corehq/apps/data_dictionary/util.py

```diff
@@ -312,8 +312,8 @@ def get_data_dict_props_by_case_type(domain, exclude_deprecated=True):
 
 
 @quickcache(vary_on=['domain'], timeout=24 * 60 * 60)
-def get_data_dict_case_types(domain):
-    case_types = CaseType.objects.filter(domain=domain).values_list('name', flat=True)
+def get_data_dict_case_types(domain, is_deprecated=False):
+    case_types = CaseType.objects.filter(domain=domain, is_deprecated=is_deprecated).values_list('name', flat=True)
```

Comment:
`is_deprecated` should be factored into the cache key, otherwise this could end up returning deprecated properties when not desired (or vice versa).

Reply:
Thank you! Addressed in d54a63831f40083baeabb51d795d8112ffafe667